docs/runbooks/VirtHandlerRESTErrorsBurst.md

```diff
@@ -2,16 +2,20 @@
 
 ## Meaning
 
-More than 80% of the rest calls failed in virt-handler for the last 5 minutes
+More than 80% of REST calls failed in `virt-handler` in the last 5 minutes.
 
-virt-handler has potentially fully lost connectivity to the apiserver. Running workloads on the affected node will not be impacted but status updates will not be propagated and actions like migrations can not take place.
+The `virt-handler` has likely fully lost connectivity to the apiserver. Running workloads on the affected node will not be impacted, but status updates will not be propagated and node-related actions, such as migrations, will fail.
 
 ## Diagnosis
 
-There exist two common types of errors:
-- The apiserver is overloaded and we run into timeouts. Issues like this can be identified by checking the apiserver metrics and looking at its response times, overall calls, …
-- The virt-handler pod can’t reach the apiserver. Common issues are: dns issues on the node, networking connectivity issues
+This error is most frequently caused by one of the following problems:
+
+- The apiserver is overloaded, which causes timeouts. To verify if this is the case, check the metrics of the apiserver, and view its response times and overall calls.
+
+- The `virt-handler` pod cannot reach the apiserver. This is commonly caused by DNS issues on the node and networking connectivity issues.
+
+Check `virt-handler` logs to identify if it can connect to the apiserver. (TBA how?)
```

Comment:
@Barakmor1 Should it be 
kubevirt logs -n <KubeVirt Install Namespace> <virt-handler-pod-name>

Should we also add before that :
kubectl get pods -n <KubeVirt Install Namespace>
To get the virt-handlers pods names?

Reply:
yes :)

Reply:
@jherrman I apologize, I see that we have a convention in other runbooks that help with getting the namespace. something like this:

- List all the the virt-handler pods
```
    $ export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
 ```
- Obtain the name of the `virt-handler` pod.
```
    $ kubectl get pods -n $NAMESPACE -l=kubevirt.io=virt-handler
```
- Review the logs for `virt-handler` and check its connectivity to the apiserver.
```
    $ kubectl logs -n  $NAMESPACE <virt-handler-pod-name>
```

Reply:
Thanks for the heads-up @sradco , should be updated now.